ansible/roles/microshift-start/tasks/main.yml

```diff
@@ -46,16 +79,40 @@
   ansible.builtin.systemd:
     name: cadvisor
     state: restarted
+  when: cadvisor_check.rc == 0
 
 - name: wait for kubeconfig to exist on microshift node
+  become: yes
   ansible.builtin.stat:
     path: "{{ remote_kubeconfig_path }}"
   register: remote_kubeconfig
   until: remote_kubeconfig.stat.exists
-  retries: 30
-  delay: 1
+  retries: 300
+  delay: 3
+
+- name: create copy of kubeconfig
+  become: yes
+  ansible.builtin.copy:
+    src: "{{ remote_kubeconfig_path }}"
+    dest: "/home/{{ ansible_user }}/.kube/config"
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+    remote_src: yes
+    force: yes
```

Comment:
kubeconfig init could also be moved to a separate task in commons, together with ~/.kube directory creation